### PR TITLE
Maps

### PR DESCRIPTION
**Pull request by KershSoftware**

### CODE REVIEW

File: config/connection.php [hunk 1]
```diff
@@ -6,4 +6,4 @@
   $q_string = "USE serverus;";
   mysqli_query($dbc, $q_string);
   
-?>
+?>
```
Comment (DeanWay) [inline config/connection.php:9]: Why is this changed?

Comment (KershSoftware) [inline config/connection.php:9]: Didn't mean to commit this but it's literally a blank line removed, nothing is changed.
